sdk/src/main/scala/io/horizen/AbstractSidechainNodeViewHolder.scala

```diff
@@ -442,7 +442,8 @@ abstract class AbstractSidechainNodeViewHolder[
 
               val stateWithdrawalEpochNumber: Int = stateAfterApply.getWithdrawalEpochInfo.epoch
               val (historyResult, walletResult) = if (stateAfterApply.isWithdrawalEpochLastIndex) {
-                val feePayments : FPI = getFeePaymentsInfo(stateAfterApply, stateWithdrawalEpochNumber)
+                val consensusEpochNumber: ConsensusEpochNumber = stateAfterApply.getCurrentConsensusEpochInfo._2.epoch
+                val feePayments : FPI = getFeePaymentsInfo(stateAfterApply, stateWithdrawalEpochNumber, consensusEpochNumber)
```

Comment:
Consider getting `consensusEpochNumber` the same way but inside the implementation of the `getFeePaymentsInfo()` method using the state passed as input parameter. In this way we can avoid modifying the signature of this abstract method (not all overridden methods really uses that value)